src/PhpSpec/CodeAnalysis/MagicAwareAccessInspector.php

```diff
@@ -33,22 +33,22 @@ public function __construct(AccessInspector $accessInspector)
      */
     public function isPropertyReadable($object, string $property) : bool
     {
-        return method_exists($object, '__get') || $this->accessInspector->isPropertyReadable($object, $property);
+        return \method_exists($object, '__get') || $this->accessInspector->isPropertyReadable($object, $property);
```

Comment:
this one won't change anything, that's always a function call.
on type-related function are replaced by better opcodes.

Reply:
more good knowledge.

Reply:
well, it still removes the need to check for a function in the current namespace (but this resolution is cached by PHP and so happens only once per function and namespace).

Reply:
@stof correct, which the benefit is not measurable in practice :)